### PR TITLE
Updated instructions.

### PR DESCRIPTION
**Pull request by emilwestergren**
@marcusnaslund peer rev

### CODE REVIEW

File: content/android/gettingstarted.md [hunk 1]
```diff
@@ -155,7 +159,7 @@ The function *setup_device* will create a copy of your camera HAL library on you
 
 <a name="PushingToPhone"></a>
 # Pushing to phone
-Every time you have rebuilt the wrapper library you can use the push function to overwrite it on the device. Make sure you set the *CAMERA_HAL* variable to the name of your original library and the correct path to your wrapper library in the setup.sh script.
+Every time you have rebuilt the wrapper library with *build* you can use the *push* function to overwrite it on the device. Make sure you set the *CAMERA_HAL* variable to the name of your original library and the correct path to your wrapper library in the *setup.sh* script.
```
Comment (marcusnaslund) [inline content/android/gettingstarted.md]: You should probably use single-quotes instead of asterisks around functions.

**Comment by marcusnaslund** [comment]
Looks good.

**Comment by emilwestergren** [comment]
Updated

**Comment by marcusnaslund** [comment]
Perfection.

**Comment by emilwestergren** [comment]
So merge?

**Comment by marcusnaslund** [comment]
Are you not allowed to?

**Comment by emilwestergren** [comment]
Yes, just wanted to make sure.

**Pull request merged**